packages/create-svelte/shared/+vitest/vite.config.js

```diff
@@ -0,0 +1,12 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+
+/** @type {import('vite').UserConfig} */
+const config = {
+	plugins: [sveltekit()],
+	test: {
+		include: ['src/**/*.{test}.{js,ts}'],
```

Comment:
I think this should be `include: ['tests/**/*.test.[t,j]s'],` from my experience in my project.